src/components/Tool/Appeal/AddAppealForm.tsx

```diff
@@ -215,13 +240,26 @@ const AddAppealForm = (): ReactElement => {
               initialGoal: 0,
```

Comment:
Remove the custom styles on the "Add Appeal" CardHeader. Maybe just keep the backgroundColor.